backend/src/main/kotlin/cz/loono/backend/notification/NotificationDefinition.kt

```diff
@@ -0,0 +1,195 @@
+package cz.loono.backend.notification
+
+import cz.loono.backend.api.dto.BadgeTypeDto
+import cz.loono.backend.api.dto.ExaminationTypeDto
+import cz.loono.backend.api.dto.SexDto
+import cz.loono.backend.api.exception.LoonoBackendException
+import cz.loono.backend.db.model.Account
+import org.springframework.http.HttpStatus
+
+object NotificationDefinition {
+
+    private const val ONESIGNAL_APP_ID = "234d9f26-44c2-4752-b2d3-24bd93059267"
+    private const val MORNING_TIME_TO_NOTIFY = "8:00AM"
+    private const val EVENING_TIME_TO_NOTIFY = "6:00PM"
+
+    fun getPreventionNotification(accounts: Set<Account>): PushNotification {
+        val name = "Prevevention notification"
+        val title = "Hola, hola, prevence volá!"
+        val text = "Mrkni, na které preventivní prohlídky se objednat."
```

Comment:
Nedáme to niekde do .properties file? Náhodne v kóde je cesta k neudržiteľnosti.

Reply:
Asi muzeme, kazdopadne ja doufam, ze to jsou jedine texty, ktere budeme udrzovat.

Reply:
Skús to teda prosím oddeliť. Nemusí byť nutne properties file, ale mať to na jednom mieste. Niečo ako notifications.properties . A tam všetky stringy. Ďakujem